homeassistant/components/sma/entities.py

```diff
@@ -0,0 +1,675 @@
+"""Entity Descriptions for the sma integration."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import (
+    STATE_CLASS_MEASUREMENT,
+    STATE_CLASS_TOTAL_INCREASING,
+    SensorEntityDescription,
+)
+from homeassistant.const import (
+    DEVICE_CLASS_ENERGY,
+    DEVICE_CLASS_POWER,
+    ELECTRIC_CURRENT_AMPERE,
+    ELECTRIC_CURRENT_MILLIAMPERE,
+    ELECTRIC_POTENTIAL_VOLT,
+    ENERGY_KILO_WATT_HOUR,
+    ENERGY_WATT_HOUR,
+    ENTITY_CATEGORY_DIAGNOSTIC,
```

Comment:
Please use the new `EntityCategory` enum instead:
`from homeassistant.helpers.entity import EntityCategory`